aws/rds_cluster.go

```diff
@@ -70,7 +69,8 @@ func nukeAllRdsClusters(session *session.Session, names []*string) error {
 			SkipFinalSnapshot:   awsgo.Bool(true),
 		}
 
-		_, err := svc.DeleteDBCluster(params)
+		print("deleting1")
```

Comment:
Leftover from testing? If not we may want to make this a more detailed logging statement.